ci.mk

```diff
@@ -60,6 +74,8 @@ pylintrc:=$(ci_dir)/.pylintrc
 pylint: $(pylintrc)
```

Comment:
should pylint also me added to non build targets? @josecm

Reply:
Solved.